Resources/public/js/views/ez-universaldiscoveryview.js

```diff
@@ -563,15 +564,17 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
                             multiple: this.get('multiple'),
                             loadContent: true,
                             isAlreadySelected: Y.bind(this._isAlreadySelected, this),
-                            startingLocationId: this.get('startingLocationId')
+                            startingLocationId: this.get('startingLocationId'),
+                            startingLocation: this.get('startingLocation'),
```

Comment:
same remark as in https://github.com/ezsystems/PlatformUIBundle/pull/799/files#r99341566, the UDW browse view should now have a startingLocation attribute definition